core/src/services/redis/backend.rs

```diff
@@ -336,6 +387,12 @@ impl kv::Adapter for Adapter {
             Capability {
                 read: true,
                 write: true,
+                // due to limitation of Redis itself,
```

Comment:
Thank you, this is the first time I've learned this. Would you like to add a note in the documents about this backend to make it clear to users?